package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "steve",
-  "version": "3.3.0",
+  "version": "3.4.0",
```

Comment:
Can you change the version numbers here + in the lockfile back to 3.3.0? This command will be included as part of 3.4.0 but there will probably be more changes before the version is incremented + released.

Reply:
changed back to 3.3.0